tests/Integration/Behaviour/Features/Scenario/Meta/meta_management.feature

```diff
@@ -0,0 +1,150 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s meta
+@reset-database-before-feature
+Feature: Meta management (Traffic & Seo)
+  PrestaShop allows BO users to manage page metadata - title, description, keywords, url_rewrite etc...
+  As a BO user I must be able to create, edit, delete and update meta data.
```

Comment:
edit and update?
And I think this behat is missing delete operations

Reply:
Yeah it does not have delete operation implemented with CQRS.  but on the other hand it would not be hard to write it but its different PR's responsability to do that IMO